infrastructure/auth.template

```diff
@@ -0,0 +1,88 @@
+{
+	"AWSTemplateFormatVersion": "2010-09-09",
+	"Description": "Template for creating resources related to authentication",
+	"Metadata": {},
+	"Parameters": {
+		"EmailARN": {
+			"Type": "String",
+			"Default": "arn:aws:ses:us-west-2:006316662763:identity/no-reply@passr.ca",
+			"Description": "SES Email Arn"
+		}
+	},
+	"Mappings": {},
+	"Conditions": {},
+	"Resources": {
+		"DevUserPool": {
+			"Type": "AWS::Cognito::UserPool",
+			"Properties": {
+				"AccountRecoverySetting": {
+					"RecoveryMechanisms": [
+						{
+							"Name": "verified_email",
+							"Priority": 1
+						}
+					]
+				},
+				"AutoVerifiedAttributes": [
+					"email"
+				],
+				"EmailConfiguration": {
+					"EmailSendingAccount": "DEVELOPER",
+					"From": "no-reply@passr.ca",
+					"ReplyToEmailAddress": "help@passr.ca",
+					"SourceArn": {
+						"Ref": "EmailARN"
+					}
+				},
+				"Policies": {
+					"PasswordPolicy": {
+						"MinimumLength": 8,
+						"RequireLowercase": true,
+						"RequireUppercase": true,
+						"TemporaryPasswordValidityDays": 7
+					}
+				},
+				"UsernameAttributes": [
+					"email"
+				],
+				"VerificationMessageTemplate": {
+					"DefaultEmailOption": "CONFIRM_WITH_LINK",
+					"EmailMessage": "Verify your email using this code: {####}\n\n If you need any assistance verifying your email, please contant help@passr.ca",
```

Comment:
That's cool we're gonna have a help email!

Reply:
Yeah, if the team wants, I can even do emails like seiya@passr.ca where it can forward to your email of choice. There is also a way to send as that email (from gmail for sure, but others might be possible)